### DISCUSSION TITLE
Translate the Rulesets' Tab in the Settings. (All languages except English)

### DISCUSSION THREAD

**Discussion by Soetch** [ppy/osu#19133] [Ideas] [answered]
## osu!
- ~~Snaking In Sliders~~ => Sliders progressifs (found this in the current version of osu!)
- ~~Snaking Out Sliders~~ => (not sure about this one, the "snaking out" term isn't really existing in French.)
- ~~Cursor Trail~~ => Traînée de curseur
- ~~Playfield Border Style~~ => Apparence de la zone de jeu
  - ~~None~~ => Sans
  - ~~Windowed~~ => Fenêtré
  - ~~Fullscreen~~ => Plein Écran

## osu!mania
- ~~Scrolling Direction~~ => Sens de Défilement
  - ~~Down~~ => Vers le bas
  - ~~Up~~ => Vers le haut
- ~~Scroll Speed~~ => Vitesse de Défilement
- ~~Timing-Based Note Colouring~~ => Coloration de Note Basée sur le Temps

![image](https://user-images.githubusercontent.com/70801309/179231512-af891e4d-c67f-4a2d-b674-c3395c05866a.png)

### Please hit me back if you have questions or modifications to propose on this translation !

**Comment by peppy** [comment, marked as answer]
please read the readme

**Reply by Soetch** [reply]
Maybe I've forgotten, yeah.
